src/deckardcain.js

```diff
@@ -1,5 +1,6 @@
 export const API_BLUEPRINT_HEADER = /^[\uFEFF]?(((VERSION:( |\t)2)|(FORMAT:( |\t)(X-)?1A))\s*?([\n\r]{1,2}|$))/i;
-export const API_BLUEPRINT_RESPONSE = /\+\s+(?:response|request)\s+\d{3}/i;
+export const API_BLUEPRINT_RESPONSE = /[-\*\+]\s+(response|request|attributes?)/i;
+export const API_BLUEPRINT_DATA = /#+\s+data structures?\s*\n/i;
```

Comment:
I think in some Markdowns you don't need the space between # and the header text. I'm not sure about our parser.

Reply:
Our parser needs the space IIRC, so we should keep it here